### ISSUE TITLE
[VersionControl] Remove all the remotes source

### ISSUE THREAD

**Issue by jsuarezruiz** [mono/monodevelop#7176]
If in a project using Git as VCS all remotes source are deleted, nothing happens.
The project is still under VCS, but without any remote.

In these cases, we can not allow to delete all the remotes (there always remains at least one); or the project is no longer under VCS.

**Comment by Therzok** [comment]
You can still operate on a repository with no remote. You're comitting and doing everything locally.

Maybe the network commands (Fetch, Push) command should show that there's no remote configured, thus nowhere to push.

**Comment by sevoku** [comment]
Yes, I agree with @Therzok , and we should show the Publish command again (in this case it won't initialize the repo, but only push it to a new remote)

**Comment by jsuarezruiz** [comment]
Perfect. So, we must disable some network commands like push, fetch, update solution, etc. and show again the publish command.

**Comment by jsuarezruiz** [comment]
Sent PR: https://github.com/mono/monodevelop/pull/7189
